--- googletest-1.8.0/Makefile.am.orig	2011-04-15 21:49:12.000000000 +0200
+++ googletest-1.8.0/Makefile.am	2011-06-16 13:08:22.272743192 +0200
@@ -209,7 +209,10 @@
   include/gtest/internal/gtest-port-arch.h \
   include/gtest/internal/gtest-string.h \
   include/gtest/internal/gtest-tuple.h \
-  include/gtest/internal/gtest-type-util.h \
+  include/gtest/internal/gtest-type-util.h
+
+pkginclude_internalcustomdir = $(pkgincludedir)/internal/custom
+pkginclude_internalcustom_HEADERS = \
   include/gtest/internal/custom/gtest.h \
   include/gtest/internal/custom/gtest-port.h \
   include/gtest/internal/custom/gtest-printers.h
@@ -288,15 +288,3 @@
 # Death tests may produce core dumps in the build directory. In case
 # this happens, clean them to keep distcleancheck happy.
 CLEANFILES = core
-
-# Disables 'make install' as installing a compiled version of Google
-# Test can lead to undefined behavior due to violation of the
-# One-Definition Rule.
-
-install-exec-local:
-	echo "'make install' is dangerous and not supported. Instead, see README for how to integrate Google Test into your build system."
-	false
-
-install-data-local:
-	echo "'make install' is dangerous and not supported. Instead, see README for how to integrate Google Test into your build system."
-	false
